Resources/public/js/views/ez-testoneview.js

```diff
@@ -0,0 +1,298 @@
+YUI().add('ez-testoneview', function(Y) {
+    'use strict';
+    /**
+     * Provides the main view of test 1
+     *
+     * @module ez-testoneview
+     */
+
+    Y.namespace('eZ');
+
+    /**
+     * Prefix for class name in TestOneView
+     *
+     * @type {string}
+     */
+    var TEST_ONE_CLASS = 'ez-testoneview';
+
+    /**
+     * The article list view
+     *
+     * @namespace eZ
+     * @class TestOneView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.TestOneView = Y.Base.create('testOneView', Y.eZ.TemplateBasedView, [], {
+        /**
+         * Builds class name with this view prefix
+         *
+         * @param {String} name
+         * @returns {String}
+         */
+        getClassName: function(name) {
+            return (typeof name === 'string' ? TEST_ONE_CLASS + '-' + name : TEST_ONE_CLASS);
+        },
+
+        /**
+         * Initialize view
+         */
+        initializer: function () {
+            var articlesContainer = this.get('articlesView').get('container'),
+                service = this.get('service');
+            this.get('articlesView').addTarget(this);
+            this._selectArticleHandler = articlesContainer.on('click', this._selectArticle(this), '[data-location-id]');
```

Comment:
Dodatkowo, `this._selectArticle(this)` jest niepoprawne. Jeśli chcesz przekazać kontekst do metody `on`, to robisz to w następujący sposób: `on(eventName, callback [this._selectArticle], context, [params])`. W przypadku powyżej powinno być wtedy tak: `articlesContainer.on('click', this._selectArticle, this, '[data-location-id]')`;